lib/config/default.js

```diff
@@ -699,7 +699,7 @@ exports.config = () => ({
      *
      * @env NEW_RELIC_CUSTOM_INSIGHTS_EVENTS_MAX_SAMPLES_STORED
      */
-    max_samples_stored: 1000
+    max_samples_stored: 30000
```

Comment:
you'll also need to PR docs https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#custom_events_max_samples_stored

Reply:
Okay done https://github.com/newrelic/docs-website/pull/9619